HeartRateMonitor.py

```diff
@@ -0,0 +1,25 @@
+class HRM:
+
+    def __init__(self, times=[], voltages=[], time_units='s', t1 = None, t2 = None):
+        self.times = times
+        self.voltages = voltages
+        self.time_units = time_units
+        self.averagehr = -1
```

Comment:
you may want to initialize this to `None`, which is the universal null or "not set" value in python (this means you don't have to rely on conventions like using -1)